userguide/virtualmachines.rst

```diff
@@ -44,85 +44,112 @@ output shows the POPCNT feature, this processor can be used with
 
-Create a virtual machine by following these steps in the creation
-wizard:
+* :guilabel:`Edit` opens a form to change the VM settings. The VM name,
+  description, number of virtual CPUs, allocated memory, boot loader,
+  and option to start the VM on system boot can all be adjusted.
 
-#. **Select VM wizard type**
+* :guilabel:`Delete` removes the VM. :ref:`Zvols <Adding Zvols>` used in
+  :ref:`disk devices <vms-disk-device>` and image files used in
+  :ref:`raw file <vms-raw-file>` devices are *not* removed when a VM
+  is deleted. These resources can be removed manually in
+  :menuselection:`Storage --> Pools` after it is determined that the
+  data in them has been backed up or is no longer needed.
 
-   Select *Virtual Machine (VM)* or
-   *Docker Host*.
+* :guilabel:`Devices` is used to add, remove, and edit the VM devices.
 
-#. **Operating System**
+* :guilabel:`Clone` makes a copy of the VM. The new clone has
+  :samp:`_clone{N}` added to the name, where :samp:`{N}` is the number
+  of the clone. %brand% displays a substantive error message if cloning
+  a VM fails.
 
-   Select the guest operating system type from the drop down menu and
-   enter an alphanumeric name for the VM. UEFI is recommended
-   as the boot method for newer operating systems.
-   :guilabel:`Start on Boot` and :guilabel:`Enable VNC` are set by
-   default. :guilabel:`Start on Boot` tells the VM to start when the
-   host system is booted. :guilabel:`Enable VNC` allows a
-   virtual connection to the VM.
+When a VM is :guilabel:`STOPPED`, the :guilabel:`Start` option becomes
+available in the |ui-options| menu. Click it to see a confirmation
+dialog that offers the option to :guilabel:`Overcommit Memory`. Memory
+overcommitment allows multiple VMs to be launched when there is not
+enough free memory for configured RAM of all VMs. Use with caution. The
+|web-ui| displays a substantive error message when a VM fails to start.
+When active, the VM :guilabel:`State` changes to :guilabel:`RUNNING`.
 
-#. **CPU and Memory**
+When a VM is :guilabel:`RUNNING`, more buttons in the |ui-options| menu
+are available:
 
-   Enter the number of virtual CPUs and the amount of memory for the
-   VM. Provide adequate memory and virtual CPUs for the applications
-   running in the VM, but be aware that providing more memory or
-   virtual CPUs than are needed can affect performance of the host
-   system.
+* :guilabel:`Power off` immediately halts the VM. This is equivalent
+  to unplugging the power cord from a computer.
 
-#. **Hard Disks**
+* :guilabel:`Stop` shuts down the VM.
 
-   Select :guilabel:`Create new disk image` to create a new zvol to use
-   as a virtual hard drive for the VM. Select
-   :guilabel:`Use existing disk image` to use an existing zvol or file
-   for the VM.
+* :guilabel:`VNC` opens a new browser tab pointed to the `VNC
+  <https://en.wikipedia.org/wiki/Virtual_Network_Computing>`__
+  port of the VM. This is used to access the VM. The VM must have a
```

Comment:
With links and markup, roughly (list requirement first):
"VMs with :guilabel:`Web Interface` enabled show a :guilabel:`VNC` button. VNC connections permit remote access to the VM.